src/server/middlewares/logger.middleware.js

```diff
@@ -0,0 +1,17 @@
+import logsService from '../services/logs.service';
+
+const loggerMiddleware = (req, res, next) => {
+  const route = req.url;
+  const time = Date.now();
+  const { method } = req;
+  let body;
+  if (method === 'GET' || method === 'DELETE') {
+    body = '';
+  } else body = JSON.stringify(req.body);
```

Comment:
you can use ternrary operator here.
`const body = method === 'GET' || method === 'DELETE' ? '' : JSON.stringify(req.body)`